blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -213,6 +250,34 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        # LE devices do not appear to expose certain properties like uuids until connect to at least once.
+        # show generic connect for these as we will not show any way to connect otherwise.
+        device_uuids = self.SelectedDevice['UUIDs']
+        show_generic_connect = not device_uuids
+        for uuid in device_uuids:
+            if uuid128_to_uuid16(uuid) in (AUDIO_SOURCE_SVCLASS_ID, AUDIO_SINK_SVCLASS_ID,HANDSFREE_AGW_SVCLASS_ID,
+                                           HEADSET_SVCLASS_ID, HID_SVCLASS_ID):
+                show_generic_connect = True
+                break
+            elif int(uuid.split("-")[0][4:6], 16) == 0x18:
+                # LE devices
+                show_generic_connect = True
```

Comment:
Forget what I wrote. I think the mouse was just sleeping when I tried to connect. 😆 `Connect` works when I make sure it's awake and changes the `Connected` and `ServicesResolved` properties.